JS/Lesson35/onur/my-app/src/components/Navbar/index.jsx

```diff
@@ -1,21 +1,138 @@
-import { NavLink } from 'react-router-dom'
+import * as React from 'react';
```

Comment:
This import add everything from react to this component but you don't event need it. Remove this line, avoid unused imports as much as possible, they slow down you application.